scripts/update-ci.sh

```diff
@@ -0,0 +1,41 @@
+#! /bin/sh
+
+set -eu
+
+mkdir -p ~/.ssh
+echo "$SSH_KEY" > ~/.ssh/id_rsa
+chmod 600 ~/.ssh/id_rsa
+ssh-keyscan github.com >> ~/.ssh/known_hosts
+
+git config --global user.email "prismabots@gmail.com"
+git config --global user.name "Prismo"
+
+git remote add github "git@github.com:$GITHUB_REPOSITORY.git"
+git fetch github "$1"
+git reset --hard "github/$1"
+git checkout "github/$1"
+
+OLD_VERSION=$(cat .github/prisma-version.txt)
+NEW_VERSION=$OLD_VERSION
+TIME_START=$(date +%s)
+
+while [ "$NEW_VERSION" = "$OLD_VERSION" ]
+do
+  if [ $(($(date +%s) - $TIME_START)) -gt 240 ]
```

Comment:
```suggestion
  # GitHub Actions workflow is scheduled every 5 minutes, so exit here after 4 minutes
  if [ $(($(date +%s) - $TIME_START)) -gt 240 ]
```

Reply:
Yes, I chose to exit earlier to avoid job overlaps which happens needlessly in the current setup sometimes